core/src/core2/indexer.clj

```diff
@@ -538,11 +538,17 @@
 
         nil))))
 
-(defn- find-fn [allocator sci-ctx scan-src tx-opts fn-id]
+(defn- find-fn [allocator sci-ctx scan-src ^ConcurrentHashMap prepare-ra-cache tx-opts fn-id]
   ;; HACK: assume xt_docs here...
   ;; TODO confirm fn-body doc key
 
-  (let [pq (op/prepare-ra '[:scan xt_docs [{id (= id ?id)} fn]])]
+  (let [lp '[:scan xt_docs [{id (= id ?id)} fn]]
+        ;; prepare-ra-cache a bit overkill here for now
```

Comment:
disagree, this is a great use of the cache - small but low-latency query, possibly used a lot :slightly_smiling_face: